doc/man3/RAND_DRBG_reseed.pod

```diff
@@ -0,0 +1,84 @@
+=pod
+
+=head1 NAME
+
+RAND_DRBG_reseed,
+RAND_DRBG_set_reseed_interval,
+RAND_DRBG_set_reseed_time_interval
+- reseed the RAND_DRBG instance
+
+=head1 SYNOPSIS
+
+ #include <openssl/rand_drbg.h>
+
+ int RAND_DRBG_reseed(RAND_DRBG *drbg,
+                      const unsigned char *adin, size_t adinlen);
+
+ int RAND_DRBG_set_reseed_interval(RAND_DRBG *drbg,
+                                   unsigned int interval);
+
+ int RAND_DRBG_set_reseed_time_interval(RAND_DRBG *drbg,
+                                        time_t interval);
+
+
+=head1 DESCRIPTION
+
+These functions are part of the OpenSSL L<RAND_DRBG(7)> API.
+
+RAND_DRBG_reseed()
+reseeds the given B<drbg>,  mixing in the specified additional data provided
```

Comment:
Remove a space?

Reply:
There seem to be various places with a double space. It's fine after a ".". (I have no idea if we ever agreed on a style for that.)

Reply:
I'll check the double spaces. As for the double spaces after the full stop: IIRC, it was Richards recommendation to avoid this question by always starting a new line after the end of a sentence. This is also more diff-friendly if a sentence needs to be changed.